_posts/2021-01-21-cockpit-236.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Cockpit 236
+author: pitti
+date: 2021-01-07
+tags: cockpit linux
+slug: cockpit-236
+category: release
+summary: Cockpit with SSH key improvements and motd editing
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 236.
+
+In the last two weeks the Cockpit authors primarily worked on some bug fixes, internal refactoring, dropping some obsolete dependencies like [jquery-flot](https://www.npmjs.com/package/jquery-flot), replacing some custom React components with standard [PatternFly](https://www.patternfly.org/v4/) ones, build system simplifications, and test stabilization. These don't have an user-visible effect, but will make Cockpit easier to develop in the future.
+
+## fslist channels: Include properties of changed files
+
+The [fslist channel API](https://github.com/cockpit-project/cockpit/blob/master/doc/protocol.md#payload-fslist1) lists files in a directory, watches for further changes to files, and sends events when changes happen. These notifications now include additional properties about the file: the owner and group name, file size, and time of last modification.
+
+This was aimed at making it more efficient to implement a file manager Cockpit UI.
```

Comment:
```suggestion
These changes were made with the goal of implementing an efficient file manager UI within Cockpit.
```

Reply:
"within Cockpit" sounds like a too strong promise -- we don't plan to build this ourselves, and certainly not inside the cockpit.git project. I changed this to a hybrid version of our proposals.